schema.ts

```diff
@@ -1064,6 +1064,68 @@ export type ObjectSchemaPathProperties = Pick<
   | 'popularityRankProperty'
 >;
 
+/**
+ * Specifies how this property should be indexed.
+ * @hidden
+ */
+export enum IndexingStrategy {
```

Comment:
Probably want to export this in index.ts so you can use it in your pack defs.

Reply:
Done.